akvo/rsr/models/organisation.py

```diff
@@ -278,13 +277,7 @@ def knowledge(self):
         def all_projects(self):
             "returns a queryset with all projects that has self as any kind of partner"
             from .project import Project
-            return Project.objects.filter(partnerships__organisation__in=self).distinct()
-
-        def public_projects(self):
-            """
-            Returns a queryset with all public projects that has self as any kind of partner.
-            """
-            return self.all_projects().public()
+            return Project.objects.of_partner(self).distinct()
```

Comment:
Ya, but what I mean is that since of_partner can take a queryset of many orgs as param doesn't it do the same thing as of_partners?